security-force-monitor/sfm-cms
# BLACKBOX: Add Sam, remove Jean and Bea, re-encrypt keys

## Overview

This PR updates the admin list to reflect current staffing. 

## Testing Instructions

* Follow the instructions to run the app: https://github.com/security-force-monitor/sfm-cms/#development
  * Confirm you can successfully decrypt the dev settings file
  * Will provide staging database dump via Slack
